Automattic/wp-calypso
# Deprecate /start/add-domain

## Proposed Changes

~This PR deprecates `/start/add-domain` flow and `/start/site-selected` flows since they are no longer in use.~ (`/start/site-selected` is not deprecated here. See the discussion below)

This PR deprecates `start/add-domain` since it's no longer in use.

## Testing Instructions



What's needed here is to confirm these two flows are not in use. There are two code references for the site-selected flow in wpcom, but they both don't look in use. By inspecting the Tracks Trend view of `calyspo_signup_start` 2fe39-pb, there is no production event with the flow prop as `add-domain` or `selected-site`. 

Please feel free to verify in some other ways; I might be overlooking something :)  

## Pre-merge Checklist
